File: singleNumbersSum.cpp
#include<iostream>
using namespace std;
int main(){
	int sum = 0;
	for(int i = 0; i < 100; i++){
		if(i % 2 == 1)
			sum += i;
	}
	cout << "Sum = " << sum;
	return 0;
}

File: singleNumbersSum.java
package springIntro;

import java.util.*;

public class singleNumbersSum {

	public static void main(String[] args) {
		int sum = 0;
		for(int i = 0; i <= 100; i++)
			if(i % 2 == 1)
				sum += i;
		System.out.println("Sum = " + sum);
	}

}
